src/hotspot/share/nmt/vmatree.hpp

```diff
@@ -37,8 +37,9 @@
 // For example, the state may go from released memory to committed memory,
 // or from committed memory of a certain MEMFLAGS to committed memory of a different MEMFLAGS.
 // The set of points is stored in a balanced binary tree for efficient querying and updating.
-class VMATree {
+class VMATree : public AnyObj {
```

Comment:
Why is it AnyObj now?

Reply:
Mainly to be able to use `new operator`. Changed to `CheapObj<mtNMT>`.